.vscode/launch.json

```diff
@@ -84,14 +84,29 @@
     },
     {
       "type": "node",
-      "name": "vscode-jest-tests",
+      "name": "vscode-jest-tests.v2",
       "request": "launch",
       "console": "integratedTerminal",
-      "internalConsoleOptions": "neverOpen",
+      "internalConsoleOptions": "openOnSessionStart",
       "cwd": "${workspaceFolder}",
-      "runtimeExecutable": "pnpm",
-      "program": "jest:vscode",
-      "args": ["--runInBand", "--watchAll=false", "--testTimeout=100000000"]
+      "runtimeExecutable": "node",
+      "runtimeArgs": ["--experimental-vm-modules"],
+      "program": "node_modules/jest/bin/jest.js",
```

Comment:
why not simply jest, like the others?

Reply:
I think the process spawn caused some weird effects for how breakpoints work, while the direct call makes debug test runs more straightforward (and it feels faster)

Reply:
Do we really need that CVE node argument for test debugging? It produces annoying message and looks unnecessary for majority of tests that someone could run from the VSCode UI.